packages/peregrine/package.json

```diff
@@ -17,7 +17,8 @@
   },
   "homepage": "https://github.com/magento/pwa-studio/tree/main/packages/peregrine#readme",
   "dependencies": {
-    "fast-glob": "~3.2.4"
+    "fast-glob": "~3.2.4",
+    "zen-observable-ts": "^1.2.0"
```

Comment:
This is not a new library. It's a dependency of Apollo Client, so we already use it indirectly.

There are better Observable implementations out there (eg, `rxjs`) but they tend to be huge. This one's free.

Reply:
That said, we ought to upgrade Apollo-related dependencies so they all resolve to the same version of this library.